tasks.sh

```
@@ -80,62 +93,65 @@ function reloadCMakeProject() {
     echo "usage: [--load] [--reload] [--clean] [--build]"
 fi
 
+# 定义操作标志
+DO_LOAD=false
+DO_RELOAD=false
+DO_CLEAN=false
+DO_BUILD=false
+RUN_EXECUTABLE=""
+
+# 第一遍：解析所有参数并设置配置
 while [[ $# > 0 ]]; do
```

Shellcheck (0.10.0)

[error] > is for string comparisons. Use -gt instead.

(SC2071)